velox/functions/prestosql/Probability.h

```diff
@@ -61,5 +62,17 @@ struct NormalCDFFunction {
   }
 };
 
+template <typename T>
+struct PoissonCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(double& result, double lambda, int64_t value) {
+    VELOX_USER_CHECK_GT(lambda, 0, "lambda must be greater than 0");
+    VELOX_USER_CHECK_GE(value, 0, "value must be a non-negative integer");
```

Comment:
The value parameter error comes before the lambda error in Java. Please can you maintain the same order so that the errors are the same if both parameters are errors in an invocation.

Reply:
Fixed it